package.json

```diff
@@ -58,6 +58,7 @@
     "ember-router-scroll": "~0.3.0",
     "ember-sanitize": "^2.0.2",
     "ember-source": "~2.14.1",
+    "ember-truth-helpers": "^1.3.0",
```

Comment:
I am not a fan of truth-helpers. I prefer a local `and` helper so as not to make the rest of the helpers from truth-helpers available.